src/ape_plugins/_cli.py

```diff
@@ -31,29 +42,65 @@ def cli():
     is_flag=True,
     help="Display all plugins (including Core)",
 )
+
+
+
 @ape_cli_context()
 def _list(cli_ctx, display_all):
-    plugins = []
+    installed_second = set()
+    installed_first = set()
+    installed_third = set()
     for name, plugin in plugin_manager.list_name_plugin():
-        version_str = ""
         version = get_package_version(name)
 
         if name in FIRST_CLASS_PLUGINS:
             if not display_all:
+            
                 continue  # NOTE: Skip 1st class plugins unless specified
-            version_str = " (core)"
+            installed_first.add(f"{name}")
 
-        elif version:
-            version_str = f" ({version})"
+        elif name in SECOND_CLASS_PLUGINS:
+            if not display_all:
+                continue
+            installed_second.add(f"{name}")
+
+        elif name not in FIRST_CLASS_PLUGINS or name not in SECOND_CLASS_PLUGINS:
+            installed_third.add(f"{name}    ({version})")
+        else:
+            print(name + "is not a plugin")
+
+    #This will most likely be empty
+    available_first = list (FIRST_CLASS_PLUGINS - installed_first)
+    
+    available_second = list (SECOND_CLASS_PLUGINS - installed_second)
+    _prefix = "ape_"
+
+    installed_first_print = _remove_prefix_add_version(installed_first, _prefix)
+    available_second_print = _remove_prefix_add_version(available_second, _prefix, version)
```

Comment:
You should define version at the top because there is a chance it won't be defined by the time we get here which will cause the program to crash

- [ ] Add `version = None` to the top of this method so it's always defined

Reply:
Ok thank you